aspnet/mvc/views/partial/sample/src/PartialViewsSample/Views/Shared/AuthorPartial.cshtml

```diff
@@ -0,0 +1,5 @@
+﻿@model string
```

Comment:
Would it be confusing to have a `@model` here and not one for `Views/Home/AuthorPartial.cshtml`?